posthog/management/commands/sync_persons_to_clickhouse.py

```diff
@@ -0,0 +1,145 @@
+        parser.add_argument(
+            "--deletes", action="store_true", help="process deletes for data in ClickHouse but not Postgres"
+        )
+        parser.add_argument("--live-run", action="store_true", help="Run changes, default is dry-run")
+
+    def handle(self, *args, **options):
+        run(options)
+
+
+def run(options, sync: bool = False):  # sync used for unittests
+    live_run = options["live_run"]
+    deletes = options["deletes"]
+
+    if not options["team_id"]:
+        logger.error("You must specify --team-id to run this script")
+        exit(1)
+
+    team_id = options["team_id"]
+
+    if options["person"]:
+        run_person_sync(team_id, live_run, deletes, sync)
+
+    if options["person_distinct_id"]:
+        run_distinct_id_sync(team_id, live_run, deletes, sync)
+
+
+def run_person_sync(team_id: int, live_run: bool, deletes: bool, sync: bool):
+    logger.info("Running person table sync")
+    # lookup what needs to be updated in ClickHouse and send kafka messages for only those
+    persons = Person.objects.filter(team_id=team_id)
+    rows = sync_execute(
+        """
+            SELECT id, max(version) FROM person WHERE team_id = %(team_id)s GROUP BY id HAVING max(is_deleted) = 0
+        """,
+        {
+            "team_id": team_id,
+        },
+    )
+    ch_persons_to_version = {row[0]: row[1] for row in rows}
+
+    for person in persons:
+        ch_version = ch_persons_to_version.get(person.uuid, None)
+        if ch_version is None or ch_version < person.version:
+            logger.info(f"Updating {person.uuid} to version {person.version}")
+            if live_run:
+                # Update ClickHouse via Kafka message
+                create_person(
+                    team_id=team_id,
+                    version=person.version,
+                    uuid=str(person.uuid),
+                    properties=person.properties,
+                    is_identified=person.is_identified,
+                    created_at=person.created_at,
+                    sync=sync,
+                )
+        elif ch_version > person.version:
+            logger.info(
+                f"Clickhouse version ({ch_version}) for '{person.uuid}' is higher than in Postgres ({person.version}). Ignoring."
+            )
+
+    if deletes:
+        logger.info("Processing person deletions")
+        postgres_uuids = {person.uuid for person in persons}
+        for uuid, version in ch_persons_to_version.items():
+            if uuid not in postgres_uuids:
+                logger.info(f"Deleting person with uuid={uuid}")
+                if live_run:
+                    create_person(
+                        uuid=str(uuid),
+                        team_id=team_id,
+                        properties={},
+                        version=int(version or 0)
+                        + 100,  # keep in sync with deletePerson in plugin-server/src/utils/db/db.ts
```

Comment:
we use this +100 in other places don't we? might be best to make it a const so we only have one place where this comment about keeping things in sync needs to live